src/libs/ReportUtils.js

```diff
@@ -1185,6 +1184,26 @@ function chatIncludesChronos(report) {
                 && _.contains(report.participants, CONST.EMAIL.CHRONOS);
 }
 
+/**
+ * @param {Object} report
```

Comment:
i think you should have an annotation for report.lastReadTime since you are using it below.

Reply:
Ah makes sense. Added.

Reply:
Ah makes sense. Added.

Reply:
Ah makes sense. Added.

Reply:
plane WiFi 😂